target/nginx/snippets/seafile-fileserver.nginx

```diff
@@ -1,13 +1,19 @@
 location /seafhttp {
-    rewrite ^/seafhttp(.*)$ $1 break;
-    proxy_pass http://127.0.0.1:8082;
-    client_max_body_size 0;
-    proxy_connect_timeout  36000s;
-    proxy_read_timeout  36000s;
-    proxy_send_timeout  36000s;
-    send_timeout  36000s;
-}
+	rewrite ^/seafhttp(.*)$ $1 break;
+	proxy_pass http://127.0.0.1:8082;
+	client_max_body_size 0;
+	proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
+
+	proxy_connect_timeout  36000s;
+	proxy_read_timeout  36000s;
+	proxy_send_timeout  36000s;
+	proxy_request_buffering off;
 
+	send_timeout  36000s;
+
+	access_log      /var/log/nginx/seafhttp.access.log seafileformat;
+	error_log       /var/log/nginx/seafhttp.error.log;
+}
 location /media {
-    root /seafile/seafile-server-latest/seahub;
```

Comment:
This commit works fine for me without the last change - I needed to keep
root /seafile/seafile-server-latest/seahub;